conda_rpms/generate.py

```diff
@@ -48,6 +49,14 @@ def render_dist_spec(dist, config):
                                 rpm_prefix=rpm_prefix,
                                 install_prefix=install_prefix)
 
+def render_env(branch_name, label, tag):
+    env_info = {'url': 'http://link/to/gh',
+                'name': branch_name,
+		'label' : label,
+                'summary': 'A SciTools environment.',
+                'version': '1',}
```

Comment:
So my worry is that there will be no difference in the version or build name for a `label` RPM despite the fact that it would be pointing to a different `tag` RPM. Did I miss something on this front?